server/src/main/java/org/opensearch/telemetry/TelemetrySettings.java

```diff
@@ -23,12 +23,26 @@ public class TelemetrySettings {
         Setting.Property.Dynamic
     );
 
+    /**
+     * Probability of sampler
+     */
+    public static final Setting<Double> TRACER_SAMPLER_PROBABILITY = Setting.doubleSetting(
+        "telemetry.tracer.sampler.probability",
+        0.001d,
```

Comment:
Please add `min` and `max` value, also `0` as default is not very useful since it would mean - no sampling at all.

Reply:
0.001d is the default value i.e we will be sampling 1% of the total request.

Reply:
Yes it is 0.1% of total request. I will be updated it to 1%.